tests/api/state/stateManager.js

```diff
@@ -55,79 +67,111 @@ tape('StateManager', (t) => {
-    )
+      st.equal(res.balance.toString('hex'), 'fff384')
 
-    st.equal(stateManager._cache._cache.keys[0], 'a94f5374fce5edbc8e2a8697c15331677e6ebf0b')
+      stateManager._cache.clear()
 
-    st.end()
-  })
+      res = await promisify(stateManager.getAccount.bind(stateManager))(
+        'a94f5374fce5edbc8e2a8697c15331677e6ebf0b',
+      )
 
-  t.test('should call the callback with a boolean representing emptiness, when the account is empty', async (st) => {
-    const stateManager = new StateManager()
+      st.equal(stateManager._cache._cache.keys[0], 'a94f5374fce5edbc8e2a8697c15331677e6ebf0b')
 
-    const promisifiedAccountIsEmpty = promisify(stateManager.accountIsEmpty.bind(stateManager), function (err, result) {
-      return err || result
-    })
-    let res = await promisifiedAccountIsEmpty('a94f5374fce5edbc8e2a8697c15331677e6ebf0b')
+      st.end()
+    },
+  )
 
-    st.ok(res)
+  t.test(
+    'should call the callback with a boolean representing emptiness, when the account is empty',
+    async st => {
+      const stateManager = new StateManager()
 
-    st.end()
-  })
+      const promisifiedAccountIsEmpty = promisify(
+        stateManager.accountIsEmpty.bind(stateManager),
+        function(err, result) {
+          return err || result
+        },
+      )
+      let res = await promisifiedAccountIsEmpty('a94f5374fce5edbc8e2a8697c15331677e6ebf0b')
 
-  t.test('should call the callback with a false boolean representing non-emptiness when the account is not empty', async (st) => {
-    const stateManager = new StateManager()
-    const account = createAccount('0x1', '0x1')
+      st.ok(res)
 
-    await promisify(stateManager.putAccount.bind(stateManager))(
-      'a94f5374fce5edbc8e2a8697c15331677e6ebf0b',
-      account
-    )
-
-    const promisifiedAccountIsEmpty = promisify(stateManager.accountIsEmpty.bind(stateManager), function (err, result) {
-      return err || result
-    })
-    let res = await promisifiedAccountIsEmpty('a94f5374fce5edbc8e2a8697c15331677e6ebf0b')
+      st.end()
+    },
+  )
+
+  t.test(
+    'should call the callback with a false boolean representing non-emptiness when the account is not empty',
+    async st => {
+      const stateManager = new StateManager()
+      const account = createAccount('0x1', '0x1')
+
+      await promisify(stateManager.putAccount.bind(stateManager))(
+        'a94f5374fce5edbc8e2a8697c15331677e6ebf0b',
+        account,
+      )
+
+      const promisifiedAccountIsEmpty = promisify(
+        stateManager.accountIsEmpty.bind(stateManager),
+        function(err, result) {
+          return err || result
+        },
+      )
+      let res = await promisifiedAccountIsEmpty('a94f5374fce5edbc8e2a8697c15331677e6ebf0b')
+
+      st.notOk(res)
 
-    st.notOk(res)
+      st.end()
+    },
+  )
 
-    st.end()
-  })
+  t.test('should generate the genesis state root correctly for mainnet', async st => {
+    if (typeof window !== 'undefined' && window.__karma__) {
```

Comment:
👍

Reply:
Good suggestion, I will add the following util:
```js
/**
 * Checks if in a karma test runner.
 * @returns {bool} is running in karma
 */
exports.isRunningInKarma = () => {
  return typeof window !== 'undefined' && window.__karma__
}
```